scripts/build.sh

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 # Build require-ale and minified assets for distribution.
+(set -o igncr) 2>/dev/null && set -o igncr; # For Cygwin on Windows compatibility
 set -o errexit
-./scripts/make-mini-d3.sh # TODO: remove
+sh scripts/make-mini-d3.sh # TODO: remove
 
 # Transpile individual files. This is useful if another module,
 # e.g. cycledash, wants to require('pileup').
 # The dist/test files are required for code coverage
 mkdir -p dist/test/{data,source,viz}
-babel src --retain-lines --ignore src/lib --out-dir dist
+node_modules/.bin/babel src --retain-lines --ignore src/lib --out-dir dist
```

Comment:
The path will not work for those who have these things installed globally, that is correct. 

On Windows it should not be run through NPM due to the ./ syntax used. That syntax is not supported by Cygwin. Instead of rewriting all the NPM scripts to play nice with Cygwin, I thought it was easier and less error prone to simply change build.sh.

As for the PATH adjustments, I'm not sure what you mean.